editor/script/cursor.gd

```diff
@@ -15,10 +58,11 @@ func _input(event):
 			
 	if event is InputEventMouseButton:
 		if event.pressed and event.button_index == 1:
-			if cursor_mesh != null:
-				var instance = cursor_mesh.scene.instantiate()
-				(instance as Node3D).position = position
-				get_parent().add_child(instance)
+			if !unplacable:
```

Comment:
Could compress this down if we don't need a specific else for either of these

`if is_placeable and cursor_mesh != null`

Reply:
Was a leftover from the `else` clause which I removed, updated 👍